### ISSUE TITLE
[Feature request] prompt before auto creation of virtual host

### ISSUE THREAD

**Issue by joostthehost** [leokhoa/laragon#57]
Would be nice to have a setting where Laragon can ask for confirmation before automatically creating a virtual host entry. For some directories I place in my document root I don't want to create a virtual host entry, while for others I do.

**Comment by leokhoa** [comment]
@joostthehost : You can remove the `auto.` prefix for your managed virtual host files.

**Comment by joostthehost** [comment]
Issue can be closed, I misunderstood part of the auto-creation process of virtual hosts 

_To clarify:
It was my understanding that virtual-hosts were also auto-created for directories that were manually placed under the document root. This appears not to be the case (at the moment)._

**Comment by leokhoa** [comment]
@joostthehost : Your understanding is right. I've released Laragon 4.0.9 which may help you (using `IgnoreList`):
https://forum.laragon.org/topic/1204/laragon-4-0-released-with-quick-add-feature?page=1

```
Added IgnoreList (laragon.ini) to skip creating virtual hosts for specified directories in DocumentRoot: https://github.com/leokhoa/laragon/issues/59

[pref]
IgnoreList=dir1|dir2|dir3
```
